lib/vfscore/automount.c

```diff
@@ -576,6 +642,25 @@ static inline int vfscore_mount_volume(const struct vfscore_volume *vv)
 		return -EINVAL;
 	}
 
+	if (vv->ukopts & LIBVFSCORE_UKOPT_MKMP)
+		uk_pr_debug("fstab: Entry with ukopt: mkmp\n");
```

Comment:
Once again, I don't think it's worth adding these `if`'s here right? They would probably be optimized out anyway by DCE though... so ok.

Reply:
The message should be printed only with this if-condition. If debug is off, the if-body becomes empty and the compiler should optimize the complete condition out.